content/en/docs/concepts/cluster-administration/events.md

```diff
@@ -0,0 +1,34 @@
+---
+reviewers:
+- piosz
+- x13n
+content_template: templates/concept
+title: Events in Kubernetes
+---
+
+{{% capture overview %}}
+
+Kubernetes events are objects that provide insight into what is happening
+inside a cluster, such as what decisions were made by scheduler or why some
+pods were evicted from the node. 
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+Since events are API objects, they are stored in the {{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}} on master. To
+avoid filling up master's disk, a retention policy is enforced: events are
```

Comment:
```suggestion
avoid filling up storage, a retention policy is enforced: events are
```
(there might be more than one server in the control plane, and it might not use its local disk for persistence)

Reply:
done - thanks! Need to **really** brush up on my Kubernetes knowledge!